Tests/ScipioKitTests/ToolchainEnvironmentTests.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
```

Comment:
You can use `@Testing` for new tests 😄

Reply:
I haven't noticed you already used it in the other tests. No problem, I'll do it next time 👍